cmd/guacone/cmd/known.go

```diff
@@ -56,6 +56,8 @@ type queryKnownOptions struct {
 	subjectType string
 	// purl / source (<vcs_tool>+<transport>) / artifact (algorithm:digest)
 	subject string
+	// search depth to search (0 is unlimited, 1 is just root node neighbors)
+	searchDepth int
```

Comment:
my only concern is the output may be too verbose and unorganized. Is there something better we can use?

Reply:
It woked for single outputs, but for a table of data we should figure out a better output

Reply:
This can be done in another PR

Reply:
issue created https://github.com/guacsec/guac/issues/1696